src/lib/Libutil/pbs_aes_encrypt.c

```diff
@@ -14,6 +14,14 @@
 extern unsigned char pbs_aes_key[];
 extern unsigned char pbs_aes_iv[];
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#define CIPHER_CONTEXT_INIT(v) EVP_CIPHER_CTX_init(v)
+#define CIPHER_CONTEXT_CLEAN(v) EVP_CIPHER_CTX_cleanup(v)
+#else
+#define CIPHER_CONTEXT_INIT(v) v = EVP_CIPHER_CTX_new(); if (!v) return -1;
+#define CIPHER_CONTEXT_CLEAN(v) EVP_CIPHER_CTX_free(v);EVP_CIPHER_CTX_free(v)
```

Comment:
Hey @mike0042 I have created [PP-880](https://pbspro.atlassian.net/browse/PP-880) to address this. Please add any details into ticket if I missed.